routes/data.js

```diff
@@ -1,57 +1,127 @@
-const {Firestore} = require('@google-cloud/firestore');
+const { Firestore } = require('@google-cloud/firestore');
 
 // Database Initialization
-const firestore = new Firestore();
-const DB_COLLECTION_NAME = (process.env.NODE_ENV == "production") ? 'movies' : 'dev-movies';
-
-exports.view = function(req, res) {
-  createMovies();
-  const movies = readMovies();
-  movies.then((data) => {
-    res.send(data);
-  })
-};
-
-// Inserts 3 new documents into the 'movies' collection with hard-coded data 
-async function createMovies() {
-  // Dummy data to insert into the database
-  const movieData = [
-    {
-      title: "Black Panther",
-      year: 2018,
-    },
-    {
-      title: "Avengers: Endgame",
-      year: 2019,
-    },
-    {
-      title: "Iron Man",
-      year: 2008,
-    }
-  ];
-  
-  movieData.forEach((movie) => {
-    let document = firestore.collection(DB_COLLECTION_NAME).doc();
-    document.set(movie);
-  });
+let firestore = new Firestore();
+
+/**
+ * Utility function that returns the correct collection name depending on whther the environment is PRODUCTION or DEVELOPMENT.
+ * @param {String} collectionName
+ */
+function resolveCollectionName(collectionName) {
+  return process.env.NODE_ENV == 'production'
+    ? collectionName
+    : 'dev-' + collectionName;
```

Comment:
Did you space this out for clarity?

Reply:
That was formatted automatically by my local prettier but the formatting has changed now.